### PR TITLE
Add my postings employer

### PR DESCRIPTION
**Pull request by IsabelleCzuboka**
Created the html and logic to post the postings from one employer to the "my postings" section on landing page. As well, I parsed the image result from employer form for it to work properly. Also, for the logic, when creating a posting, the employer ID gets added to the posting in database

### CODE REVIEW

**Review by Guillaume-Lachapelle — APPROVED** [review]
Nice work! Tested with NASA employer and it works.

File: src/app/job-post/job-post.component.ts
Comment (Holyvert) [inline src/app/job-post/job-post.component.ts:1]: this was needed

**Review by Holyvert — APPROVED** [review]

**Review by Guillaume-Lachapelle — APPROVED** [review]
Changed if in job-post line 76 to comply with ng test. Please always do "ng test" before opening a pull request.

**Review by MillerSty — APPROVED** [review]

**Review by KarinaSandur — APPROVED** [review]
Great work!

**Pull request merged**